tests/system.py

```diff
@@ -383,6 +384,50 @@ def test_managing_subscription_iam_policy(
     assert bindings[1].members == ["group:cloud-logs@google.com"]
 
 
+def test_subscriber_not_leaking_open_sockets(
+    publisher, topic_path, subscriber, subscription_path, cleanup
+):
```

Comment:
Seems like it would be clearer not to use the `subscriber` feature for this systest, but rather create separate instances for both the context-manager version and the cleanup one.